pkg/ctl/create/cluster.go

```diff
@@ -131,6 +131,9 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 	if cfg.Metadata.Version == "" || cfg.Metadata.Version == "auto" {
 		cfg.Metadata.Version = api.DefaultVersion
 	}
+	if cfg.Metadata.Version == "latest" {
+		cfg.Metadata.Version = api.LatestVersion
+	}
```

Comment:
this is the only place where this seems to matter, like how we accept `version: ""` for falling back to the default version